packages/discord.js/src/structures/interfaces/InteractionResponses.js

```diff
@@ -225,6 +234,19 @@ class InteractionResponses {
     return options.fetchReply ? this.fetchReply() : undefined;
   }
 
+  /**
+   * Presents a modal component
+   * @param {APIModal|ModalData|Modal} modal The modal to present
+   */
```

Comment:
```suggestion
   * Shows a modal component
   * @param {APIModal|ModalData|Modal} modal The modal to show
   */
```

Reply:
These are pretty much the same, going to keep this as-is for now.